packages/app/src/cli/models/extensions/specifications/validation/app_config_webhook.ts

```diff
@@ -13,14 +14,41 @@ export function webhookValidator(schema: object, ctx: zod.RefinementCtx) {
 function validateSubscriptions(webhookConfig: WebhooksConfig) {
   const {subscriptions = []} = webhookConfig
   const uniqueSubscriptionSet = new Set()
-  const uniqueComplianceSubscriptionSet = new Set()
 
   if (!subscriptions.length) return
 
+  if (
+    webhookConfig.privacy_compliance &&
+    webhookConfig.subscriptions?.some((subscription) => subscription.compliance_topics)
+  ) {
+    return {
+      code: zod.ZodIssueCode.custom,
+      message: `The privacy_compliance section can't be used if there are subscriptions including compliance_topics`,
+    }
+  }
+
+  const ComplianceTopics = subscriptions.flatMap((subscription) => subscription.compliance_topics).filter(Boolean)
+  if (uniq(ComplianceTopics).length !== ComplianceTopics.length) {
+    return {
+      code: zod.ZodIssueCode.custom,
+      message: 'You can’t have duplicate subscriptions with the same compliance topic',
```

Comment:
```suggestion
      message: 'You can’t have multiple subscriptions with the same compliance topic',
```

but I'd run this by UX

Reply:
Ah yes agreed - I was wondering if we had made a decision I missed previously since we'd been using "duplicate" before, sorry I didn't make that clear! Deffo will await Jen's opinion here :)

Reply:
I think either works! I'm cool with multiple in this example, and duplicate in the other, we can have some variation in how we explain it because we're crystal clear about the actual issue (same compliance topic)